docs/design-language.md

```diff
@@ -223,6 +223,22 @@ A few guidelines should be followed to ensure that the list element looks unifor
 
 When a hover state is triggered, it cannot just rely on a colour change. There also must be a state change that shows the differentiation. For example, in our programs, there is a background colour change, as well as the addition of a side border colour. Hover states always include a styling change, and the background colour will get a gradient.
 
+## Loading
+
+Two options are available to denote a loading state, the "3 dot loader" and "Skeleton". Skeleton is the preferred method with the 3 dots being phased out where applicable.
+
+### Skeleton
+
+This loader can be customized to display a placeholder of incoming content. This helps improve perceived responsiveness. When using this method it is best to create a simplified layout, just enough to give the user an idea of whats coming. Animate skeleton using the "wave" style. Read more about skeleton in the **<a href="https://mui.com/material-ui/react-skeleton/" target="_blank">MUI docs</a>.**
+
+<img src="/img/loaders/skeleton-loader.gif" alt="3 dot loader animation" title="3 dot loader animation" className="img-example" />
+
+### 3 Dot Loader
+
+The 3 dot animated loader is the original loading animation throughout VertiGIS Studio apps. This is gradually being replaced by skeleton and should not be used in new projects.
```

Comment:
We should talk about when and where the 3 dot loader is still appropriate.

Reply:
Updated.

Reply:
I'm not sure if the audience reading this cares that this was the original loading animation.

Reply:
Another use case is when you are loading something that you don't know what it will return or it will take an indeterminate amount of time to load something.

Reply:
> I'm not sure if the audience reading this cares that this was the original loading animation.

I think it's worth mentioning at this time as devs may come across it and wonder why skeleton hasn't been used instead.

Reply:
What are the high level rules we want to use for choosing a loader. The Analytics gif shows an example of what not to do. Here are some ideas for discussion:
------
When to use a 3 dot loader
1. Large blocks/regions of a layout are loading
2. If the content loading is unknown and the format, size, shape, or layout could vary.

When to use a Skeleton
1. Smaller pieces of content that can be represented with basic geometric shapes

Reply:
Updated. For the sake of getting this out there I temporarily removed the example until Analytics is updated.

Reply:
Looks good